nixos/lib/make-options-doc/default.nix

```diff
@@ -1,20 +1,106 @@
-/* Generate JSON, XML and DocBook documentation for given NixOS options.
+/**
+  Generates documentation for nix modules.
 
-   Minimal example:
+  # Outputs
 
-    { pkgs,  }:
+  This function offers the following outputs
```

Comment:
Should we be more specific about the type of this? "Returns an attrset with the following keys" or something? Functions having multiple outputs isn't really a thing at language level.